### DISCUSSION TITLE
disableStatusPagesInDevelopment = false doesn't work

### DISCUSSION THREAD

**Discussion by Nikolastual** [adonisjs/core#1474] [Help]
Hi!

Custom pages 500..599 work with the exception handler **but not 404 error pages**
I am linking to a post from another AdonisJS user on: 
[https://stackoverflow.com/questions/63156539/cant-render-404-page-in-adonisjs-development-environment](https://stackoverflow.com/questions/63156539/cant-render-404-page-in-adonisjs-development-environment)

```
import Logger from '@ioc:Adonis/Core/Logger'
import HttpExceptionHandler from '@ioc:Adonis/Core/HttpExceptionHandler'

export default class ExceptionHandler extends HttpExceptionHandler {
	protected disableStatusPagesInDevelopment = false
	protected statusPages = {
		'404': 'errors/not-found',
		'500..599': 'errors/server-error'
	}

	constructor () {
		super(Logger)
	}
}
```
Config:
NODE_ENV = development
404 File:
resources/views/errors/not-found.edge

**Comment by thetutlage** [comment]
No need to create multiple threads for a single issue. I will look into it.

Duplicate of https://github.com/adonisjs/core/discussions/1473
